hxtool/cli/gpslog.py

```diff
@@ -141,8 +141,8 @@ def write_json(log_data: bytes, file_name: str) -> int:
         logger.warning("Log is blank. Not writing empty JSON log")
         return 0
     jlog = {
-        "header": hexlify(bytes(log.header)).decode("ascii"),
-        "unknown": hexlify(log.unknown_3c).decode("ascii"),
+        "header": hexlify(bytes(log.sectors[0].header)).decode("ascii"),
+        "unknown": hexlify(log.sectors[0].unknown_3c).decode("ascii"),
```

Comment:
I wasn't sure why the raw log header bytes were included in the JSON output, so I left the header of the first sector to not change this part of the output. If something more appropriate is warranted, pls let me know.

Reply:
It's been a while, but if I remember correctly, the intention of including all the data was to get bijective conversion between the two formats.

Reply:
OK. I'm not sure how to do this with logs having more than one sector — the sector header information seems to be specific to the raw layout... I could add all the sector headers here but I'm not sure it'd be very helpful. Is what the code is doing here OK or should it do something else? Thanks.